### PR TITLE
Update workflows

### PR DESCRIPTION
**Pull request by ocean90**
**Description**
* Updates and simplifies used actions
* Fixes new issues reported by PHPStan
* Bumps PHP requirements because `szepeviktor/phpstan-wordpress` requires PHP 7.2, see https://github.com/wearerequired/traduttore/runs/6833736663?check_suite_focus=true

**How has this been tested?**

**Screenshots**
If applicable, add screenshots to show the visual change.

**Types of changes**
Breaking change

**Checklist:**
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code has proper inline documentation.

### CODE REVIEW

**Comment by ocean90** [comment]
@swissspidy Do you maybe have an idea why the `@require-php-7.4` tag for the behat tests isn't working?

**Comment by swissspidy** [comment]
@ocean90 Looks like wp-cli-tests' `behat-tags.php` looks for tests in the `features/` folder by default, which breaks the tags.

This can be addressed by setting the `` environment variable though:

https://github.com/wp-cli/wp-cli-tests/blob/e387388712add62664278aeb89517734fe5e13fd/utils/behat-tags.php#L43

So this works:

```bash
BEHAT_FEATURES_FOLDER=tests/features composer behat
```

File: composer.json [hunk 1]
```diff
@@ -44,12 +44,13 @@
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
     "php-stubs/wp-cli-stubs": "^2.4",
     "phpunit/phpunit": "^7.5.20",
```
Comment (swissspidy) [inline composer.json:46]: Maybe worth using `yoast/wp-test-utils` here or changing the version constraint to ensure packages can be installed on PHP 8.x

**Comment by ocean90** [comment]
> So this works:
> 
> ```shell
> BEHAT_FEATURES_FOLDER=tests/features composer behat
> ```

@swissspidy Thanks for the hint. Looks like that and https://github.com/wearerequired/traduttore/pull/236/commits/30b68ece7a19a5b28440ae921693b13dcf613cf8 did fix it.

**Comment by swissspidy** [comment]
> > So this works:
> > ```shell
> > BEHAT_FEATURES_FOLDER=tests/features composer behat
> > ```
> 
> @swissspidy Thanks for the hint. Looks like that and [30b68ec](https://github.com/wearerequired/traduttore/commit/30b68ece7a19a5b28440ae921693b13dcf613cf8) did fix it.

IIRC I used `vendor/bin/behat` there because there were otherwise issues with code coverage gathering. But since that is currently not working anyway, should be fine for now.

Though it would be nice to fix the code coverage eventually :)

**Review by swissspidy — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (swissspidy) [reply composer.json:46]: Created #237 for this

**Pull request merged**